src/utils/gist.ts

```diff
@@ -13,17 +13,9 @@
  * @returns {(string | null)}
  */
 export function getGistId(rawInput: string): string | null {
-  let input = rawInput.trim();
+  const id = rawInput.trim().match(/[0-9A-Fa-f]{32}/);
 
-  let id = input;
-  if (input.startsWith('https://gist.github.com')) {
-    if (input.endsWith('/')) {
-      input = input.slice(0, -1);
-    }
-    id = input.split('/').pop()!;
-  }
-
-  return id.match(/[0-9A-Fa-f]{32}/) ? id : null;
+  return id?.[0] || null;
 }
```

Comment:
question: do we have unit tests for this function?

Reply:
Yes. Here: https://github.com/electron/fiddle/blob/3814ec9be6e432eca26eaade778efd15d0968c48/tests/utils/gist-spec.ts#L41-L85

Reply:
It'd be nice if the null case(s) were covered in the testsuite.

Reply:
Good idea.

Reply:
Done